README.rst

```diff
@@ -1,9 +1,27 @@
 |logo| Pootle
 =============
 
-|chat| |build| |health| |coverage| |requirements|
+.. image:: https://img.shields.io/gitter/room/translate/pootle.svg?style=flat-square
+    :alt: Join the chat at https://gitter.im/translate/pootle
+    :target: https://gitter.im/translate/pootle
+
+.. image:: https://img.shields.io/travis/translate/pootle/master.svg?style=flat-square
+    :alt: Build Status
+    :target: https://travis-ci.org/translate/pootle
+
+.. image:: https://landscape.io/github/translate/pootle/master/landscape.svg?style=flat-square
+    :target: https://landscape.io/github/translate/pootle/master
+    :alt: Code Health
+
+.. image:: https://img.shields.io/coveralls/translate/pootle/master.svg?style=flat-square
+   :target: https://coveralls.io/github/translate/pootle?branch=master
+   :alt: Test Coverage
+
+.. image:: https://img.shields.io/requires/github/translate/pootle.svg?style=flat-square
+   :target: https://requires.io/github/translate/pootle/requirements/?branch=master
+   :alt: Requirements
+
 
-.. Resources: 
```

Comment:
Does this comment prevent rendering? It was another one of those lets show this for users who won't se the rendering, without it there isn't much context.  But not a blocker